aztk/spark/models/plugins/jupyter_lab/configuration.py

```diff
@@ -0,0 +1,23 @@
+import os
+from aztk.models.plugins.plugin_configuration import PluginConfiguration, PluginPort, PluginRunTarget
+from aztk.models.plugins.plugin_file import PluginFile
+from aztk.utils import constants
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+class JupyterLabPlugin(PluginConfiguration):
+    def __init__(self):
+        super().__init__(
+            name="jupyter_lab",
```

Comment:
Sorry to be super pedantic, but I think this might be better as `jupyterlab` since they brand it as one word. I think we should also make plugins case insensitive (not in this pr, just in general).